drivers/python/rethinkdb/net_twisted.py

```diff
@@ -20,17 +20,6 @@
 pResponse = p.Response.ResponseType
 pQuery = p.Query.QueryType
 
-def start_timeout(timeout, *defersToFail):
```

Comment:
This was used in two places, but one place needed an absolute timeout rather than a relative timeout, which is a slightly different code path - I initially broke this up into two functions, but since each was only used in exactly one place, it didn't seem worth it to have a generic function.